client/v3/patch_cluster.go

```diff
@@ -0,0 +1,24 @@
+package clientv3
```

Comment:
This is placed in a separate file to avoid having merge conflicts with upstream changes.